Lib/test/test_httpservers.py

```diff
@@ -823,6 +831,28 @@ def test_path_without_leading_slash(self):
         self.assertEqual(response.getheader("Location"),
                          self.tempdir_name + "/?hi=1")
 
+    def test_extra_headers_list_dir(self):
+        with mock.patch.object(self.request_handler, 'extra_response_headers', new=[
+            ('X-Test1', 'test1'),
+            ('X-Test2', 'test2'),
+        ]):
+            response = self.request(self.base_url + '/')
+            self.assertEqual(response.getheader("X-Test1"), 'test1')
+            self.assertEqual(response.getheader("X-Test2"), 'test2')
+
+    def test_extra_response_headers_get_file(self):
+        with mock.patch.object(self.request_handler, 'extra_response_headers', new=[
```

Comment:
```suggestion
        with mock.patch.object(self.request_handler, 'extra_response_headers', [
```

no need for using the 'new' keyword (same for the previous test)

Reply:
Fixed in e99780e1f429089ff09be0bf5d1e0011342006f7